cric-exchange-app-server/package-lock.json

```diff
@@ -0,0 +1,1183 @@
+{
```

Comment:
Do we generally publish package-lock.json for node projects?

Reply:
Ideally you always want to publish the lock file to get consistent builds. https://stackoverflow.com/questions/63187000/what-is-the-npm-equivalent-of-yarn-install-frozen-lockfile